enaml/qt/docking/q_dock_container.py

```diff
@@ -92,7 +93,7 @@ def _closestDockBar(container):
 
     tie_breakers = (3, 0, 2, 1)
     borders = (QDockBar.North, QDockBar.East, QDockBar.South, QDockBar.West)
-    values = zip(edge_ranks, edge_weights, tie_breakers, borders)
+    values = list(zip(edge_ranks, edge_weights, tie_breakers, borders))
```

Comment:
`list` isn't necessary here. Alternatively use the in-place sort method over sorted

Reply:
Indeed.